pull-images/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Pull images"
+description: "pull images down from docker to prevent extra builds"
+inputs:
+  baseTarget:
+    required: false
+    type: string
+  solrTarget:
+    required: false
+    type: string
+  subdir:
+    required: false
+    type: string
+  webTarget:
+    required: false
+    type: string
+  workerTarget:
+    required: false
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Pull base image to prevent build
+      if: ${{ inputs.baseTarget != '' }}
+      shell: bash
+      run: >-
+        cd ${{ inputs.subdir }};
+        docker compose pull base &&
+        echo 'Pulled $TAG'
+    - name: Pull solr image to prevent build
+      if: ${{ inputs.solrTarget != '' }}
+      shell: bash
+      run: >-
+        cd ${{ inputs.subdir }};
+        docker compose pull solr &&
+        echo 'Pulled $TAG'
```

Comment:
Love the echo, can we take it one step further and add the image type?
Example:
```yaml
echo 'Pulled solr $TAG'
```